ydb/core/tx/datashard/sample_k.cpp

```diff
@@ -0,0 +1,336 @@
+#include "scan_common.h"
+#include "datashard_impl.h"
+#include "upload_stats.h"
+#include "range_ops.h"
+
+#include <ydb/core/base/appdata.h>
+#include <ydb/core/base/counters.h>
+#include <ydb/core/scheme/scheme_tablecell.h>
+#include <ydb/core/tablet_flat/flat_row_state.h>
+#include <ydb/core/kqp/common/kqp_types.h>
+
+#include <ydb/core/tx/tx_proxy/proxy.h>
+#include <ydb/core/tx/tx_proxy/upload_rows.h>
+
+#include <ydb/library/yql/public/issue/yql_issue_message.h>
+
+#include <ydb/core/ydb_convert/ydb_convert.h>
+#include <util/generic/algorithm.h>
+#include <util/string/builder.h>
+#include <ydb/core/ydb_convert/table_description.h>
```

Comment:
```suggestion
#include "datashard_impl.h"
#include "range_ops.h"
#include "scan_common.h"
#include "upload_stats.h"

#include <ydb/core/base/appdata.h>
#include <ydb/core/base/counters.h>
#include <ydb/core/kqp/common/kqp_types.h>
#include <ydb/core/scheme/scheme_tablecell.h>
#include <ydb/core/tablet_flat/flat_row_state.h>
#include <ydb/core/tx/tx_proxy/proxy.h>
#include <ydb/core/tx/tx_proxy/upload_rows.h>
#include <ydb/core/ydb_convert/table_description.h>
#include <ydb/core/ydb_convert/ydb_convert.h>
#include <ydb/library/yql/public/issue/yql_issue_message.h>

#include <util/generic/algorithm.h>
#include <util/string/builder.h>
```

Reply:
Ok, but I really want to use clang-format for such things.
Done